orocos_toolchain/CHANGELOG.rst

```diff
@@ -0,0 +1,209 @@
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Changes, New Features, and Fixes for the Orocos Toolchain
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Toolchain 2.9
+=============
+
+The Orocos Toolchain v2.9 release series mainly improved on the
+correct execution of the Component updateHook() and allowing
+extended configuration options when connecting Data Flow ports.
+
+There was also a big change in the RTT StateMachine execution
+semantics on a cycle-by-cycle basis, see the Caveats.
+
+Important Caveats
+-----------------
+
+* updateHook() will now only be executed when an 'user' triggered
+  event has happened, and no longer when internal bookkeeping
+  of the ExeuctionEngine happens. For full detail, see PR
+  https://github.com/orocos-toolchain/rtt/pull/91
```

Comment:
Mention that the ActivityInterface class changed and all activity implementations have to be modified, too (e.g. orocos/rtt_ros_integration#53).